editor/js/Menubar.File.js

```diff
@@ -279,6 +279,20 @@ Menubar.File = function ( editor ) {
 	} );
 	options.add( option );
 
+	// Export STL (Binary)
+	
+	var option = new UI.Row();
+	option.setClass( 'option' );
+	option.setTextContent( 'Export STL (Binary)' );
+	option.onClick( function () {
+
+		var exporter = new THREE.STLExporter();
+
+		saveString( exporter.parse( editor.scene, { binary: true } ), 'model-binary.stl' );
```

Comment:
Shouldn't this use `saveArrayBuffer()` instead?

Reply:
Oh, yes we should use `saveArrayBuffer()` instead for binary format. I'll change it now. : )